api/optimeet/activities/Activities.csv

```diff
@@ -0,0 +1,37 @@
+Category,SubCategory,Google API Key

```

Comment:
Can we delete this file since we have the JSON one?

Reply:
done